### PR TITLE
Added market order function

### PR DESCRIPTION
**Pull request by Abhinavcode13**
Added the function

### CODE REVIEW

**Comment by Abhinavcode13** [comment]
Please check now @ighoshsubho

**Comment by ighoshsubho** [comment]
The logic is commented in the main.py file

**Comment by Abhinavcode13** [comment]
@ighoshsubho updated it :)

**Comment by Abhinavcode13** [comment]
@ighoshsubho check once again

File: helpers/functions.py [hunk 1]
```diff
@@ -1822,3 +1822,25 @@ def free_cash_flow_to_equity(
...
+
+import datetime
+
+def execute_market_order(order_type, symbol, quantity):
+    # Simulating order execution
+    current_price = get_current_price(symbol)  # Function to get the current price for the symbol
+    timestamp = datetime.datetime.now()
+
+    if order_type == "buy":
+        execute_buy_order(symbol, current_price, quantity)  # Function to execute a buy order
```
Comment (ighoshsubho) [inline helpers/functions.py:1837]: Where is execute_buy_order function??

File: helpers/functions.py [hunk 2]
```diff
@@ -1822,3 +1822,25 @@ def free_cash_flow_to_equity(
...
+        print(f"Buy order for {quantity} shares of {symbol} executed at market price {current_price} at {timestamp}")
+    elif order_type == "sell":
+        execute_sell_order(symbol, current_price, quantity)  # Function to execute a sell order
+        print(f"Sell order for {quantity} shares of {symbol} executed at market price {current_price} at {timestamp}")
+    else:
+        print(f"Invalid order type: {order_type}")
```
Comment (nikhil25803) [inline helpers/functions.py:1843]: Instead of printing the value, do return the response so that it can be used elsewhere.

File: main.py [hunk 3]
```diff
@@ -3072,3 +3072,21 @@ def free_cash_flow_to_equity(
...
+    
+    
+@app.get(
+    "/market_order",
+    tags=["market_order"],
+    description="Execute a market order",
+)
+def market_order(symbol: str, quantity: int, side: str):
+    try:
+        return {
```
Comment (nikhil25803) [inline main.py:3084]: You are just accepting the values and returning the same value without any processing and calculation.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]
Make some required changes and do upload a SS of the endpoint response next time as well.

**Comment by ighoshsubho** [comment]
Please check the CONTRIBUTIONS.md and make a fresh PR

**Pull request closed without merging**